src/shared/types.ts

```diff
@@ -0,0 +1,26 @@
+import { DispatchProp } from 'react-redux';
+
+interface IScaling {
+  min: number;
+  max: number;
+}
+
+interface INodePoolStatus {
+  nodes: number;
+  nodes_ready: number;
+}
+
+export interface INodePool {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  availability_zones: any;
+  id: string;
+  name: string;
+  node_spec: object;
+  scaling: IScaling;
+  status: INodePoolStatus;
+}
+
+export interface IDispatchProps extends DispatchProp {
```

Comment:
It's common practice to name props coming from `mapDispatchToProps` `IDispatchProps`, and this name kind of collides with that. Maybe rename to something more action-related?

Reply:
👍